modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,249 @@
+Use this option if you want to reuse a SSH key that was used for SSH-Push Salt clients on the server.
+
+Create and Populate Configuration Channel rhn_proxy_config_1000010001?::
+Accept default ``Y``.
+
+SUSE Manager Username::
+Use same user name and password as on the {productname} server.
+
+If parts are missing, such as CA key and public certificate, the script prints commands that you must execute to integrate the needed files.
+When the mandatory files are copied, run [command]``configure-proxy.sh`` again.
+If you receive an HTTP error during script execution, run the script again.
+
+[command]``configure-proxy.sh`` activates services required by {productname} Proxy, such as [systemitem]``squid``, [systemitem]``apache2``, [systemitem]``salt-broker``, and [systemitem]``jabberd``.
+
+To check the status of the proxy system and its clients, click the proxy system's details page on the {webui} (menu:Systems[Proxy], then the system name).
+[guimenu]``Connection`` and [guimenu]``Proxy`` subtabs display various status information.
+
+
+
+[[uyuni-proxy-setup-pxe]]
+== Enable PXE Boot
+
+
+
+[[proxy.pxe.sync]]
+=== Synchronize Profiles and System Information
+
+To enable PXE boot through a proxy, additional software must be installed and configured on both the {susemgrproxy} and the  {productname} Server.
+
+. On the {susemgrproxy} install [package]#susemanager-tftpsync-recv#:
++
+
+----
+zypper in susemanager-tftpsync-recv
+----
+
+. On the {productname} Proxy, run the [command]``configure-tftpsync.sh`` setup script and enter the requested information:
++
+
+----
+configure-tftpsync.sh
+----
++
+
+It asks for hostname and IP address of the {productname} Server and of the proxy itself.
+Additionally, it asks for the tftpboot directory on the proxy.
+
+. On the {productname} Server, install [package]``susemanager-tftpsync``:
++
+
+----
+zypper in susemanager-tftpsync
+----
+
+// same file name on the server and proxy!
+. On the {productname} Server, run [command]``configure-tftpsync.sh`` to configure the upload to the {susemgrproxy}:
++
+
+----
+configure-tftpsync.sh FQDN_of_Proxy
+----
+
+. To start an initial synchronization on the {productname} Server run:
++
+
+----
+cobbler sync
+----
++
+
+It can also be done after a change within Cobbler that needs to be synchronized immediately.
+Otherwise Cobbler synchronization will run automatically when needed.
+For more information about Cobbler, see xref:client-configuration:cobbler.adoc[Cobbler].
+
+
+
+[[proxy.pxe.dhcp]]
+=== Configure DHCP for PXE through Proxy
+
+{productname} is using Cobbler to provide provisioning.
+PXE (tftp) is installed and activated by default.
+To enable systems to find the PXE boot on the {susemgrproxy} add the following to the DHCP configuration for the zone containing the systems to be provisioned:
+
+----
+next-server: <IP_Address_of_Proxy>
+filename: "pxelinux.0"
+----
+
+
+////
+// REMARK: 2019-08-23, ke: this needs closer checking
```

Comment:
We should check this, as now we are duplicating the question for SUSE Manager 4.1 and Uyuni (and 4.0 if we port there)

Reply:
new: https://github.com/SUSE/spacewalk/issues/10521

Reply:
I suggest you add the issue URL here.